opencti-platform/opencti-front/src/private/components/entities/organizations/OrganizationEditionOverview.tsx

```diff
@@ -125,11 +154,11 @@ const OrganizationEditionOverviewComponent = (props) => {
     });
   };
 
-  const handleSubmitField = (name, value) => {
+  const handleSubmitField = (name: string, value: string | string[] | number | number[] | null) => {
```

Comment:
value might be an `Option` as I see in line 161, so in the list of possible types we should add `Option`, so you don't have then to cast with `as unknown`

Reply:
done, thank you